packages/react-flex/Spec.md

```diff
@@ -0,0 +1,524 @@
+| Stack              | `verticalFill`         | Redundant. Same as a style override of `height: 100%`.                                                                                                                                   |
+| Stack/Flex         | `padding`              | Already extended from `HTMLElement`, we can avoid redundancy by leveraging the native prop here.                                                                                         |
+| Flex               | `space`                | Given that `horizontalAlign` and `verticalAlign` are abstractions of `align-items` and `justify-content` respectively, spacing can now be defined through them instead of overriding it. |
+| Flex               | `debug`                | Only helpful for development phases, extra layer of maintenance with a small added benefit to the user. User can alternatively use style overrides for the same effect.                  |
+| Flex               | `fill`                 | Redundant. Same as a style override of `height: 100%; width: 100%`.                                                                                                                      |
+
+### FlexItem
+
+| Name     | Type                                                                                            | Default value | Comments                                                                                                                                                 |
+| -------- | ----------------------------------------------------------------------------------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `align`  | string (_see [align-self](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self#values)_) | "`auto`"      | Abstraction of `align-self`.                                                                                                                             |
+| `grow`   | boolean \| number \| string = "`inherit`", "`initial`", "`unset`"                               | `0`           | Abstraction of `flex-grow` and override of Flex's `grow`. A `true` value will translate into `1` for ease of use and retro-compatibility with Stack.     |
+| `shrink` | boolean \| number \| string = "`inherit`", "`initial`", "`unset`"                               | `1`           | Abstraction of `flex-shrink` and override of Flex's `shrink`. A `true` value will translate into `1` for ease of use and retro-compatibility with Stack. |
+| `push`   | boolean                                                                                         | `false`       | Defines an auto margin depending on the flex direction. Kept for retro-compatibility but perhaps can be removed.                                         |
```

Comment:
+1 to remove it

Reply:
It does not make sense to me why this would be a feature but I'll wait to see if there is a reason for this prop's existence. If there are none, I'll deprecate it as it can easily be achieved with some style overrides like `marginTop: auto` / `marginLeft: auto` like the original implementation.

Reply:
As mentioned to @khmakoto, will remove this